src/content/guidelines/motion/resources.md

```diff
@@ -6,7 +6,7 @@ tabs: ['Basics', 'Choreography', 'Resources']
 
 <anchor-links>
 <ul>
-    <li><a href="#motion-generator">Motion generator</a></li>
+    <li><a href="#motion-generator">Motion Generator</a></li>
```

Comment:
should this be capitalized?

Reply:
Lemme check with Clay one more time.

Reply:
Clay gave the thumb up! 
![image](https://user-images.githubusercontent.com/15144993/52661272-4a370480-2ec7-11e9-9d55-f8aa6bb5f285.png)